html/index.html

```diff
@@ -0,0 +1,259 @@
+	   t[1] = document.getElementById("demo1").children[0].src;
+	   t[2] = document.getElementById("demo2").children[0].src; 
+	   t[3] = document.getElementById("demo3").children[0].src;
+	   t[4] = document.getElementById("demo4").children[0].src; 
+	   t[5] = document.getElementById("demo5").children[0].src;
+	   t[6] = document.getElementById("demo6").children[0].src;
+	   t[7] = document.getElementById("demo7").children[0].src,
+	   t[8] = document.getElementById("demo8").children[0].src;
+	  
+       	
+	  if(count >= 4 && ((t[0] == t[1] && t[1] == t[2] && t[0] != d)  ||
+				        (t[0] == t[3] && t[3] == t[6] && t[0] != d)  ||  
+					    (t[0] == t[4] && t[4] == t[8] && t[0] != d)  ||
+					    (t[1] == t[4] && t[4] == t[7] && t[1] != d)  ||  
+					    (t[2] == t[5] && t[5] == t[8] && t[2] != d)  ||  
+					    (t[2] == t[4] && t[4] == t[6] && t[2] != d)  ||   
+					    (t[3] == t[4] && t[4] == t[5] && t[3] != d)  || 
+					    (t[6] == t[7] && t[7] == t[8] && t[6] != d)))       
+					
+		           	{	
+					   if(global_key == "x") 
+					   {  
+					      b++;
+					      s++;
+						  if(s==1){
+						  document.getElementById("spn").innerHTML = s;
+						  }
+						  else{
+						  document.getElementById("spn").innerHTML = s;
+						  document.getElementById("time1").innerHTML = "<b>times</b>";
+						  } 
+						  document.getElementById("win").innerHTML = ' ';
+						  document.getElementById("msg").style.display = "inline-block";
+						  document.getElementById("reset1").style.display = "inline-block";
+				          document.getElementById("msg1").innerHTML = "<b>X has won the Game!!</b>"+"<br/>"+"<b>Play Again!!</b>";
+					   }
+					   else if(global_key == "o") 
+					   {
+					      b++;
+					      a++;
+						  if(a==1){
+						  document.getElementById("spn1").innerHTML = a;
+						  }
+						  else{
+						  document.getElementById("spn1").innerHTML = a;
+						  document.getElementById("time3").innerHTML = "<b>times</b>";
+						  } 
+						  document.getElementById("win").innerHTML = ' ';
+						  document.getElementById("msg").style.display = "inline-block";
+						  document.getElementById("reset1").style.display = "inline-block";
+						  document.getElementById("msg1").innerHTML = "<b>0 has won the Game!!</b>"+"<br/>"+"<b>Play Again!!</b>"; 
+						}
+                    }
+	  
+				//draw logic
+	  else if(count == 9 )
+	    {    
+		    c++;
+			if(c==1){
+		    document.getElementById("spn2").innerHTML = c;
+			 }
+			 else{
+			document.getElementById("spn2").innerHTML = c;
+		    document.getElementById("time5").innerHTML = "<b>times</b>";
+		    } 
+		    document.getElementById("win").innerHTML = ' ';
+		    document.getElementById("msg").style.display = "inline-block";
+			document.getElementById("reset1").style.display = "inline-block";
+		    document.getElementById("msg1").innerHTML = "<b>Ohh!! it's a draw!!</b>" 
+			
+		} 
+		
+ }   
+	    //reset logic
+	  function myFunction(){
```

Comment:
Give meaningful names to functions , variables , classes and ids . Right now some names do not show it's functionalities and usages. Ex. myFunction , class 'one' , and some variables like var 'd'.
In myFunction instead of accessing each div by its id you can access it using class name.